modules/core/resources/META-INF/smithy/proto/proto.smithy

```diff
@@ -47,10 +48,23 @@ enum protoNumType {
 /// PROTOBUF indicates that the default encoding should be used
 /// EPOCH_MILLIS indicates that an int64 should be used instead of the
 /// default encoding.
+/// RFC3339_STRING indicates that the timestamp should be encoded as a string
+/// using the format described in RFC3339 section 5.6.
 @trait(selector: ":test(timestamp, member > timestamp)")
 enum protoTimestampFormat {
   PROTOBUF
   EPOCH_MILLIS
+  RFC3339_STRING
+}
+
+/// Specifies how enumeration value should be encodeds in protobuf.
+/// ORDINAL indicates that the enum value should be encoded as an integer.
+/// STRING_VALUE indicates that the enum value should be encoded as the
+/// string value.
+@trait(selector: ":test(enum, member > enum)")
+enum protoEnumFormat {
+  ORDINAL
+  STRING_VALUE
 }
```

Comment:
Can you elaborate on when one would use an enum shape with an `ORDINAL` format rather than using an `intEnum` shape?

Reply:
Ah I see, I didn't realize that was the case, that makes more sense then thanks

Reply:
I updated the selector to not allow the trait to be applied to open enumerations.